test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,118 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+void LoadBalancerFuzzBase::initializeASingleHostSet(uint32_t num_hosts_in_host_set,
+                                                    uint8_t index_of_host_set) {
```

Comment:
nit: suggest a more clear name: priority instead of index_of_host_set to remind reader this corresponds to a priority level

Reply:
Changed. Good idea. This also prompted me to change it around the whole class.